### PR TITLE
Filter out overly old tweets

### PR DESCRIPTION
**Pull request by xyzshantaram**
Previously, searching using the app did not do any sort of date-based refinement and sometimes results included extremely old tweets, such as from 2015 in some cases. This commit aims to fix that by introducing a maximum age for displayed tweets.

### CODE REVIEW

**Comment by ucalyptus2** [comment]
@shantaram3013 makes sense. Good idea.

**Comment by xyzshantaram** [comment]
Hi, @umanghome, bump, could you please review this and either close or merge it?

File: App.svelte [hunk 1]
```diff
@@ -382,6 +395,12 @@
           </label>
         </div>
 
+        <div>
+          <label for="dayCount"><strong>Show only tweets from the last</strong></label>
+          <input type="number" bind:value={inputs.days} id="dayCount" min=1/>
```
Comment (viksit) [inline App.svelte]: @shantaram3013 also makes sense to add hours, given that the expiry period for this information is in the hours/minutes?

Comment (xyzshantaram) [reply App.svelte]: ~Does twitter allow filtering by last n hours?~

Edit: It does not.

**Comment by umanghome** [comment]
Something like this makes more sense, unchecked by default:

- [ ] Show tweets only from the last `4` hours

Where `4` is a tiny input field. Filter gets applied only if the checkbox is selected.

**Comment by xyzshantaram** [comment]
> Something like this makes more sense, unchecked by default:
> 
>     * [ ]  Show tweets only from the last `4` hours
> 
> 
> Where `4` is a tiny input field. Filter gets applied only if the checkbox is selected.

Twitter does not have n-hours search, as far as i can tell.

**Comment by xyzshantaram** [comment]
> > Something like this makes more sense, unchecked by default:
> > ```
> > * [ ]  Show tweets only from the last `4` hours
> > ```
> > 
> > 
> > Where `4` is a tiny input field. Filter gets applied only if the checkbox is selected.
> 
> Twitter does not have n-hours search, as far as i can tell.

However, I've implemented the change in the style you suggested (and of course, updated this to reflect the UI changes introduced in 9969d12). Please review / advise / hopefully merge.

**Comment by zeryx** [comment]
Is this ready for review @umanghome?

**Pull request closed without merging**